pom.xml

```diff
@@ -221,14 +222,53 @@
         </plugins>
       </build>
     </profile>
+      <profile>
+      <id>test-cpp</id>
+      <activation>
+        <property>
+          <name>BUILD_TESTS</name>
+          <value>ON</value>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>${antrun.version}</version>
+            <executions>
+              <execution>
+                <id>test-sparkrapidsjni</id>
+                <phase>test</phase>
+                <configuration>
+                  <target>
+                    <exec dir="${project.basedir}" 
```

Comment:
It's optional , just wanted to point out that there is an exec-related issue #386 . If you  happen to create a reusable ant script wrapping exec, that we can call with `<ant >` task , we can reuse it to fix it consistently everywhere in this pom.